bridge/config.go

```diff
@@ -1,7 +1,9 @@
 package bridge
 
+// Config represents the Elrond Config parameters
 type Config struct {
-	NetworkAddress string
-	BridgeAddress  string
-	PrivateKey     string
+	NetworkAddress       string
+	BridgeAddress        string
+	PrivateKey           string
+	NonceUpdateInSeconds uint
```

Comment:
Please also update the `cmd/config.toml.example` and we should consider a sensibel default for this.

Reply:
already done, please check the value in this PR
https://github.com/ElrondNetwork/elrond-eth-bridge/blob/b9f6b46b9ae037f52067045019341eb28d61c512/cmd/config.toml.example#L10

Reply:
added an example (60 seconds)